render.yaml

```diff
@@ -0,0 +1,36 @@
+# v0.7.4bis
+services:
+    - type: web
+      name: nango-server
+      env: docker
+      repo: https://github.com/NangoHQ/nango-render
+      plan: starter
```

Comment:
I think there are some [free plans](https://render.com/pricing#compute) available, both for the service & the db. We should probably use this by default.

Reply:
Here are the options in the Render Blueprint Specs -> https://github.com/NangoHQ/nango/issues/318